src/QirRuntime/lib/QIR/context.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <assert.h>

```

Comment:
Minor:
Typically the C++ files do not include the C standard headers _directly_ (`#include <assert.h>`) but instead the C++ files include the C++-wrappers of those (`#include <cassert>`). I don't know exactly why.

Reply:
Would you mind filing a bug in this repo, something along the lines of "QirRuntime coding style improvements" for this and maybe other issues you can think of while looking at the code? I'm absolutely open to doing the cleanups and standardizing on better conventions but those changes are better to be done across the project and not mixed into the feature PRs. Thank you!